plugin.go

```diff
@@ -47,6 +47,7 @@ type (
 		Parameters []*Parameter
 		dispatchFunc
 		PluginCanDoFunc
+		PluginCloseFunc
```

Comment:
Seems like I missed that in #62 as well: there's a separate type `Dispatcher` which was intended to embed all plugin dispatch calls. Could you please move `PluginCanDoFunc` and `PluginCloseFunc` there?